### ISSUE TITLE
ERROR GENERATING CODE

### ISSUE THREAD

**Issue by wwzeze** [abi/screenshot-to-code#377]
**Describe the bug**
the error message came out when generating code using image--to-code
**To Reproduce**
1. when to the localhost site
2. upload images
3. and the error message came out

**Screenshots of backend AND frontend terminal logs**
![Screenshot 2024-07-19 215506](https://github.com/user-attachments/assets/1389e596-ecf7-4fd4-abdb-e8078e85d6d9)
![Screenshot 2024-07-19 215729](https://github.com/user-attachments/assets/26487747-314f-499b-bd95-cf48202deaba)
![Screenshot 2024-07-19 215740](https://github.com/user-attachments/assets/04e66ff6-2560-4de2-a92a-8b24fc810df4)

**Comment by abi** [comment]
Looks like your front-end and back-end are set up correctly. Can you trying going to localhost:7000 and let me know what you see? seems like the problem is with your backend and front-end connecting to each other.

**Comment by cybrking** [comment]
Getting same error.

**Comment by wwzeze** [comment]
![Screenshot 2024-07-25 180938](https://github.com/user-attachments/assets/09ffa5a2-9aa4-4ef2-ad28-3f5348dbaa8f)
@abi here is it,localhost 7000,loosk normal and working

**Comment by abi** [comment]
Hmm... are your front-end and backend running on the same machine? I'm not sure from the information why they're not able to connect.

**Comment by wwzeze** [comment]
yes,my front-end and back-end running on the same machine,but with different command prompt tab.

**Comment by cybrking** [comment]
I even get this error when running as container.  Any news on this issue?

**Comment by abi** [comment]
@Ali91Geyik please open a new issue and describe what you've tried and share screenshots. Hard for me to debug this issue  without enough information.
